src/css/flags/controls-disabled.less

```diff
@@ -10,4 +10,8 @@
     .jw-media {
         cursor: auto;
     }
+
```

Comment:
Don't you also need to do this to `.jw-captions` for captions rendered in a div (Firefox)?

```css
.jw-captions {
    max-height: none;
}
```